plugins/okta/komand_okta/tasks/monitor_logs/schema.py

```diff
@@ -44,7 +44,7 @@ class MonitorLogsOutput(insightconnect_plugin_runtime.Output):
   "title": "Logs",
   "description": "All system logs within the specified time range",
   "items": {
-    "type": "object"
+    "$ref": {}
```

Comment:
I think we need to keep this schema untouched or it can cause issues when the output changes in the task